lib/Quantization/Quantization.cpp

```diff
@@ -163,10 +163,11 @@ class FunctionQuantizer : public FunctionConverter {
   Node *createConversion(Function &function, NodeValue &val,
                          TypeRef destTy) override {
     if (destTy->isQuantizedType()) {
-      return function_.createQuantize("quantize", val, destTy);
+      return function.createQuantize("quantize", val, destTy);
     }
-    assert(destTy->getElementType() == ElemKind::FloatTy && "");
-    return function.createDequantize("quantize", val);
+    assert(destTy->getElementType() == ElemKind::FloatTy &&
+           "Can't dequantize to any type except float.");
```

Comment:
Can you add explicitly that it's fp32, we also support fp16, but not for dequantize.